### DISCUSSION TITLE
skhd modes

### DISCUSSION THREAD

**Discussion by tradiff** [asmvik/yabai#1906] [General]
Previously I was using the border color of the focused window as and indicator of the current mode. (swap mode, resize mode, warp mode, etc).

With [window borders being removed](https://github.com/koekeishiya/yabai/commit/5dd912f617ae8d5e2f2a27a84201ae6b4b5dc7b7), I'm wondering what other people who enjoy using modes with yabai are doing for feedback of which mode you're in.

**Comment by gldtn** [comment]
Perhaps @xiamaz can add something to his Yabai Indicator? He did mentioned that he might be adding more features to it.

https://github.com/koekeishiya/yabai/discussions/1111
